### PR TITLE
Pull in  the latest ursula-monitoring plugins

### PR DESCRIPTION
**Pull request by lihkin213**
pull in the inspec sensu plugin.

### CODE REVIEW

**Comment by nirajdp76** [comment]
LGTM

File: envs/example/defaults-2.0.yml [hunk 1]
```diff
@@ -584,8 +584,8 @@ common:
     mcelog: False
   ursula_monitoring:
     method: tar # git|tar
-    tar_url: https://file-mirror.openstack.blueboxgrid.com/ursula-monitoring/2.2.5.tar.gz
-    tar_version: 2.2.5
+    tar_url: https://file-mirror.openstack.blueboxgrid.com/ursula-monitoring/2.2.11.tar.gz
+    tar_version: 2.2.11
```
Comment (paulczar) [inline envs/example/defaults-2.0.yml]: we should probably switch this to master for dev

Comment (nirajdp76) [inline envs/example/defaults-2.0.yml]: is master kept current?

Comment (lihkin213) [inline envs/example/defaults-2.0.yml]: @paulczar Did you want me to switch this to pull in the master branch of ursula-monitoring?

Comment (nirajdp76) [inline envs/example/defaults-2.0.yml]: no just, https://file-mirror.openstack.blueboxgrid.com/ursula-monitoring/master.tar.gz

**Comment by epwn** [comment]
retest

**Comment by epwn** [comment]
retest

**Comment by lihkin213** [comment]
retest

**Comment by lihkin213** [comment]
Pointed this to the ursula-monitoring master branch. Please review and merge.

**Pull request merged**